src/pyj/read_book/iframe.pyj

```diff
@@ -323,9 +323,9 @@ class IframeBoss:
     def on_next_screen(self, data):
         backwards = data.backwards
         if current_layout_mode() is 'flow':
-            flow_scroll_by_page(-1 if backwards else 1)
+            flow_scroll_by_page(-1 if backwards else 1, data.flip_if_rtl_page_progression)
```

Comment:
Added this new value to the data since we don't know here whether or not we should flip.